ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,59 @@
+<!--
+Compensation requests have a specific format that enables them to be evaluated by a bot for more efficient budgeting. Please make sure you stick to the template. A linter will evaluate your request after you remove [WIP] from the issue title and notify you of any issues.
+
+See the following link for details:
+https://bisq.wiki/Making_a_compensation_request
+
+Feel free to browse other requests if you're not sure about something, or reach out in #compensation on Keybase (https://keybase.io/team/bisq).
+-->
+
+
 ## Summary
 
-> _Specify the total amount of BSQ you are requesting:_
+> _Specify the total amount of BSQ you are requesting, along with the USD total and BSQ/USD rate:_
 
- - BSQ requested:
- 
- <!--
- Don't forget to make your compensation request in the actual Bisq DAO! See the following link for details:
- https://docs.bisq.network/compensation.html
- -->
+- **BSQ requested**: **`[BSQ amount]`** <!-- [BSQ amount] should be: USD amount / BSQ rate -->
+- USD requested: [USD amount] <!-- [USD amount] should be: total of all contributions listed in "contributions delivered" table below -->
+- BSQ rate: [rate] USD per BSQ <!-- [rate] should be the rate used to determine the BSQ amount above, as specified in the cycle's rate announcement (https://bisq.wiki/Compensation_Maintainer#Announce_BSQ-USD_rate) -->
 
```

Comment:
```suggestion
- Last request: # 

```

Reply:
@jmacxx would such a change require any modifications to bot code?

Reply:
I don't see any impact to the bot by that change.  Presumably the user would link back to their previous compensation request.